CHANGELOG.md

```diff
@@ -10,6 +10,66 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 -   Session claims related types/classes and the validateClaims function
+-   Added `getInvalidClaimsFromResponse` to the SessionClass to help parsing responses with invalid claim errors
+-   Added `API_INVALID_CLAIM` event to the Session recipe
+
+### Breaking changes
+
+-   Backend SDKs have to be updated first to a version that supports session claims before enabling EmailVerification!
+    -   supertokens-node: 12.0
+-   EmailVerification recipe is now not initialized as part of auth recipes. You can add it to the recipe list as `EmailVerification.init` like other recipes.
+-   Removed `verifyEmail`, `sendVerificationEmail` and `isEmailVerified` from auth recipes. These should now be called on the `EmailVerification` recipe
+-   Moved email verification related events, overrides, pre-api hooks and redirection contexts into the `EmailVerification` recipe. You should configure them while initializing the `EmailVerification` recipe.
+-   Fix typing of `consumeCode` in the passwordless recipe
```

Comment:
This is an older fix.. not sure why this is here. Also, the changelog you have modified if way older.. the current version of web-js is `0.1.3`. Please update your branches